main.py

```diff
@@ -428,7 +422,9 @@ def execute_simulations(
         output_manager.set_metadata_prefix(metadata_file["prefix"])
         is_data_valid = input_manager.start_data_processing(str(metadata_file["path"]), True)
         if is_data_valid:
-            output_manager.add_log("Validation complete", "Data is valid. \nSimulating...\n", info_map)
+            output_manager.add_log(
+                "Validation complete", f"Data is valid. \nSimulating...{metadata_file['prefix']}" " scenario", info_map
```

Comment:
```suggestion
                "Validation complete", f"Data is valid. Simulating {metadata_file['prefix']} scenario.", info_map
```
I think the log message looks a little cleaner like this.

Reply:
Updated!